spec/features/applications/new_application_spec.rb

```diff
@@ -40,14 +40,15 @@
                     :phone_number => '2077020720', 
                     :description => 'I will be a good pet owner, trust me.'}
     # check_box_tag where you select pet(s), select one pet for now (@pet_1) to make sure the other is left on /favorites
-    within("span#pet_#{@pet_1.id}") do
-      check "pet_ids_"
-    end
-
+    
   end
   
   it 'you can apply for a pet that will then remove that pet from the favorites page' do
     
+    within("span#pet_#{@pet_1.id}") do
```

Comment:
copied the selection of the first `pet` to each of the tests to allow the last test to have no `pet` selected